apps/cookbook/src/app/examples/stock-chart-example/stock-chart-example.component.ts

```diff
@@ -4,12 +4,22 @@ import { Options } from 'highcharts';
 
 import { StockChartDataPoint, stockChartOptions } from '@kirbydesign/designsystem';
 
-@Component({
+const config = {
   selector: 'cookbook-stock-chart-example',
-  templateUrl: './stock-chart-example.component.html',
-  styleUrls: ['./stock-chart-example.component.scss'],
-})
+  template: `
+    <kirby-stock-chart
+      description="Accessibility description goes here"
+      [options]="options"
+      [data]="data"
+      [height]="height"
+    >
+    </kirby-stock-chart>
+  `,
```

Comment:
Whitespace is significant in code viewer - please align:
```suggestion
  template: `<kirby-stock-chart
  description="Accessibility description goes here"
  [options]="options"
  [data]="data"
  [height]="height">
</kirby-stock-chart>`,
```

Reply:
Done.